packages/std/src/math/decimal.rs

```diff
@@ -618,6 +618,12 @@ impl<'de> de::Visitor<'de> for DecimalVisitor {
     }
 }
 
+impl PartialEq<&Decimal> for Decimal {
+    fn eq(&self, rhs: &&Decimal) -> bool {
+        self == *rhs
+    }
+}
```

Comment:
Wh, can we implement that using `forward_ref_binop!` too? That should give us 3 implementations: reference left, right and both

Reply:
Doesn't seem so. `forward_ref_binop!` looks for an `Output` associated type (like when you're trying to do this for `Add`), but `PartialEq` doesn't have such a type.

Reply:
We also don't need an implementation for refs on both sides. [That case is taken care of by deref coercion.](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=f31b7e7eccd44940d18e35dc48ecd09b)

Reply:
Ah, good to know, thanks @uint. Can you add tests for all combinations and the missing implementation (maybe as PR into this PR)?

Reply:
Sure thing. Let's get `1.1.0` wrapped up.